src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/ClientMock.swift

```diff
@@ -152,7 +152,7 @@ extension ClientMock: Client {
 			return
 		}
 
-		onGetRegistrationToken(forKey, withType, isFake, completeWith)
+		onGetRegistrationToken(forKey, withType, dateOfBirthKey, isFake, completeWith)
```

Comment:
Did I missed something in my PR?

Reply:
Same as above, only needed for some of my tests :)